feature-libs/asm/styles/components/_asm-main-ui.component.scss

```diff
@@ -134,12 +124,10 @@
       }
     }
 
-    @include forVersion(5.1) {
-      color: inherit;
-      background-color: #ffffff;
-      box-shadow: 2px 2px rgba(85, 107, 130, 0.1),
-        inset 0px -1px 0px rgba(85, 107, 130, 0.2);
-    }
+    color: inherit;
+    background-color: #ffffff;
```

Comment:
Is there a reason why asm doesn't use the color variables?

Reply:
Hi @ercultimate, Do you know this reason?

Reply:
ASM components use a separate UI theme from the Spartacus storefront, so it should not be using the storefront theme variables.